core/pom.xml

```diff
@@ -179,6 +179,13 @@
             <artifactId>javax.ws.rs-api</artifactId>
         </dependency>
 
+        <!-- JDK 11 Changes -->
+        <dependency>
+            <groupId>javax.xml.bind</groupId>
+            <artifactId>jaxb-api</artifactId>
+            <scope>runtime</scope>
+        </dependency>
```

Comment:
also.. since this was depracted in jdk9 doesn't this mean the jdk runtime will not supply it anymore? 
 If true, then shouldn't this be a compile time explicit dependency (vs `runtime`)? 

I may be reading this wrong.
http://openjdk.java.net/jeps/320

Lemme know if I am grok'ing this incorrectly.

Reply:
Was able to avoid using the flag by simply updating the maven plugin versions to latest and plumb-in the latest `org.ow2.asm:asm:7.0` as dependency.

Reply:
Moved `<scope>runtime</scope>` to parent `pom.xml`